model/resource/host.yaml

```diff
@@ -5,133 +5,32 @@ groups:
     brief: >
         A host is defined as a computing instance. For example, physical servers, virtual machines, switches or disk array.
     attributes:
-      - id: id
-        type: string
-        brief: >
-          Unique host ID. For Cloud, this must be the instance_id assigned by the cloud provider.
-          For non-containerized systems, this should be the `machine-id`. See the table below for
-          the sources to use to determine the `machine-id` based on operating system.
-        examples: ['fdbf79e8af94cb7f9e8df36789187052']
-      - id: name
-        type: string
-        brief: >
-          Name of the host. On Unix systems, it may contain what the hostname
-          command returns, or the fully qualified hostname, or another name
-          specified by the user.
-        examples: ['opentelemetry-test']
-      - id: type
-        type: string
-        brief: >
-          Type of host. For Cloud, this must be the machine type.
-        examples: ['n1-standard-1']
-      - id: arch
-        type:
-          allow_custom_values: true
-          members:
-            - id: amd64
-              value: 'amd64'
-              brief: "AMD64"
-            - id: arm32
-              value: 'arm32'
-              brief: "ARM32"
-            - id: arm64
-              value: 'arm64'
-              brief: "ARM64"
-            - id: ia64
-              value: 'ia64'
-              brief: "Itanium"
-            - id: ppc32
-              value: 'ppc32'
-              brief: "32-bit PowerPC"
-            - id: ppc64
-              value: 'ppc64'
-              brief: "64-bit PowerPC"
-            - id: s390x
-              value: 's390x'
-              brief: "IBM z/Architecture"
-            - id: x86
-              value: 'x86'
-              brief: "32-bit x86"
-        brief: >
-            The CPU architecture the host system is running on.
-      - id: image.name
-        type: string
-        brief: >
-          Name of the VM image or OS install the host was instantiated from.
-        examples: ['infra-ami-eks-worker-node-7d4ec78312', 'CentOS-8-x86_64-1905']
-      - id: image.id
-        type: string
-        brief: >
-          VM image ID or host OS image ID. For Cloud, this value is from the provider.
-        examples: ['ami-07b06b442921831e5']
-      - id: image.version
-        type: string
-        brief: >
-          The version string of the VM image or host OS as defined in
-          [Version Attributes](README.md#version-attributes).
-        examples: ['0.1']
-      - id: ip
-        type: string[]
+      - ref: host.id
```

Comment:
reference to the `host.name` got lost here

Reply:
thanks, fixed it, also updated with recent changes in host as well